src/main/java/training/RobotTypes.java

```diff
@@ -0,0 +1,7 @@
+package training;
+
+public enum RobotTypes {
```

Comment:
```suggestion
public enum RobotTypes {

```

formatting: add space here